guides/integrate-nextauth.md

```diff
@@ -0,0 +1,327 @@
+
+### Create an Identity
+
+Creating a user can be done either in the admin console or through an API. This guide will use the admin console. Navigate to your realm in the console and click **Identities**, then click **Add identity**.
+
+For more information visit [Workflows: User and Group Provisioning](/docs/v1/workflows/user-provisioning).
+
+<Arcade clip={Clip.CreateIdentity} />
+
+### Generate a passkey
+
+Once you have an identity you can generate a passkey. This step can also be done either in the admin console or through an API. This guide will use the admin console. Navigate back to **Identities** and select the identity you would like to bind to a passkey. Click **Add a passkey**, select your app and the click **Proceed & send email**. The user will receive an enrollment email which they can tap on to bind a passkey to their device.
+
+For more information visit [Workflows: Bind Passkey To User](/docs/v1/workflows/bind-passkey).
+
+<Arcade clip={Clip.CreatePasskey} />
+
+### Bind passkey to device
+
+Once the user taps on the enrollment email, they will be redirected to your application. Intercept the link from the enrollment email. The link that is redirected to your application will take on the following form. A `/bind` path will be appended to your Invoke URL (configured in your application above) as well as several other query parameters.
+
+```
+$invoke_url/bind?api_base_url=<api_base_url>&tenant_id=<tenant_id>&realm_id=<realm_id>&identity_id=<identity_id>&job_id=<job_id>&token=<token>
+```
+
+Once you receive the incoming URL, pass it into the SDK to complete the binding process. You can validate the incoming URL with `isBindPasskeyUrl`. Upon success, a private key will have been created in the device's hardware trust module and the corresponding public key will have been sent to the Beyond Identity Cloud. At this point the user has a passkey enrolled on this device.
+
+```javascript
+if (embedded.isBindPasskeyUrl(url)) {
+  const bindPasskeyResponse = await embedded.bindPasskey(url);
+}
+```
+
+## Authenticate with Passkey
+
+Once you have one passkey bound to a device, you can use it to authenticate. For more information visit [Workflows: Authenticate with Passkey](/docs/v1/workflows/authentication).
+
+### Craft an Authorization URL
+
+First you will need to craft an authorization URL. The base url can be found in the Beyond Identity Admin Console
+under your application, select "EXTERNAL PROTOCOL". Copy the `Authorization Endpoint` and add the following additional query parameters:
+
+![Authorize Url](../docs/workflows/screenshots/authentication-auth-url.png)
+
+```bash title="/authorize"
+https://auth-$REGION.beyondidentity.com/v1/tenants/$TENANT_ID/realms/$REALM_ID/applications/$APPLICATION_ID/authorize?
+response_type=code
+&client_id=$APPLICATION_CLIENT_ID
+&redirect_uri=$REDIRECT_URI
+&scope=openid
+&state=$STATE
+&code_challenge_method=256
+&code_challenge=$PKCE_CODE_CHALLENGE
+```
+
```

Comment:
I don't think you actually need to direct the developer to craft an URL here because I think the NextAuth provider figures this out for them through the `wellKnown` url, right?

Reply:
True. Done.